src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.cs

```diff
@@ -51,36 +54,38 @@ internal partial class FormatCommandHandler :
             _editorOperationsFactoryService = editorOperationsFactoryService;
         }
 
-        private void Format(ITextView textView, Document document, TextSpan? selectionOpt, CancellationToken cancellationToken)
+        private async Task Format(ITextView textView, Document document, TextSpan? selectionOpt, CancellationToken cancellationToken)
```

Comment:
Task returning methods should end in 'Async'

Reply:
reverted